package.json

```
@@ -23,7 +23,9 @@
     "typeorm": "typeorm-ts-node-commonjs -d datasource.ts",
     "typeorm:migrations": "typeorm migration:run -d dist/datasource.prod.js",
     "pregrpc:types": "rimraf src/grpc/gen",
-    "grpc:types": "proto-loader-gen-types --longs=String --enums=String --keepCase --defaults --oneofs --grpcLib=@grpc/grpc-js --outDir=src/grpc/gen/ src/grpc/protos/posts.proto src/grpc/protos/users.proto"
+    "grpc:types": "proto-loader-gen-types --longs=String --enums=String --keepCase --defaults --oneofs --grpcLib=@grpc/grpc-js --outDir=src/grpc/gen/ src/grpc/protos/posts.proto src/grpc/protos/users.proto",
+    "deps": "docker-compose up",
```

`docker compose up -d` -> to start as daemon.
Also, version 2 of docker compose uses `docker compose` instead of `docker-compose`.

```suggestion
    "deps": "docker compose up -d",
```